pkg/helm/helmchart_structure_exists.go

```diff
@@ -98,7 +98,7 @@ func (r *HelmStructureExists) Test() *tractusx.QualityResult {
 	return &tractusx.QualityResult{Passed: true}
 }
 
-func isChartDirectory(dir string) bool {
+func IsChartDirectory(dir string) bool {
```

Comment:
Is there a use case, that already needs that as public function in the library?
If so, I would add Godoc describing the function.

Reply:
Yes, dashboard uses the function to check if directory is a helm chart. 
Added comment

Reply:
Ok fine with me. Although you could argue, that a "contains Chart.yaml" could be implemented in the dashboard itself instead of coupling to another library method :)